deployment/docker/kafka-puller/run_app.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+bootstrap_server="$1"
+topic_name="$2"
+./gradlew build
```

Comment:
Seems like we should be doing the build in the dockerfile not here

Reply:
+1